tests/registration/test_registration.py

```diff
@@ -13,35 +15,136 @@
     name="(!) WAIT LINK. Description of the tested functionality",
 )
 class TestRegistration:
-    @title("User registration under minimum requirements")
-    @description(
-        "WHEN the user submits the minimum required data for registration, "
-        "THEN the information about the user is stored in the database"
-    )
-    def test_of_registration(self, postgres):
-        with step("Generation data for registration"):
-            email = generate_string(length=2, additional_characters=["@te.st"])
+    """
+    Test suite for testing the registration of a new user
+    """
+    email = None
+    user_to_register = None
+
+    def setup_method(self):
+        """
+        Generate data for registration and user registration
+        """
+        with step("Generation data for registration and user registration"):
+            self.email = generate_string(length=10, additional_characters=["@te.st"])
             first_name = generate_string(length=2)
             last_name = generate_string(length=2)
             password = generate_string(
                 length=8, additional_characters=["@1"]
             ).capitalize()
 
-            body = RegistrationSteps().data_for_sent(
-                email=email,
+            self.user_to_register = RegistrationSteps().data_for_sent(
+                email=self.email,
                 first_name=first_name,
                 last_name=last_name,
                 password=password,
             )
 
+    @severity(severity_level="CRITICAL")
+    @title("User registration under minimum requirements")
+    @description(
+        "WHEN the user submits the minimum required data for registration, "
+        "THEN the information about the user is stored in the database"
+    )
+    def test_of_registration(self, postgres):
+        """
+        Test of registration of user with minimum requirements
+        Args:
+            postgres: fixture for working with the database
```

Comment:
It is not necessary to add a description to the tests. Title and description should explain everything

Reply:
Fixed